packages/shell-api/src/field-level-encryption.ts

```diff
@@ -75,17 +82,17 @@ export class ClientEncryption extends ShellApiClass {
       value,
       {
         keyId: encryptionId,
-        algorithm: encryptionAlgorithm
+        algorithm: encryptionAlgorithm as any
       }
     );
   }
 
   @returnsPromise
-  decrypt(
+  async decrypt(
     encryptedValue: any
-  ): Promise<BinaryType> {
+  ): Promise<any> {
```

Comment:
Shouldn't this stay Promise<Binary>?

Reply:
Inside `mongodb-client-encryption` it's not guaranteed to be `Binary`, at least reading through the code and looking at the tests in there it can just come out as a string. One of the tests stuffs a `"hello"` into `encrypt(...)`, then `decrypt(...)`s that result, to verify it matches `"hello"` again which obviously isn't a binary 😅

Reply:
Huh, that's odd! We can just use `Document` then instead of `any`, for consistency.

Reply:
@addaleax I'd be fine to change it to `Document` if you want - but the API in itself also just works on strings for example... See also the official "encrypt" example: https://docs.mongodb.com/manual/reference/method/ClientEncryption.encrypt/#example. It just passes in a string so when I'd decrypt that I'd get back my string right?

Reply:
yeah, I think that’s fine then :) just wanted to make sure